tests/test_parameterspace.py

```diff
@@ -703,5 +703,23 @@ def test_fix_with_inactive_constants():
     assert not space.has_conditions()
 
 
+def test_integer_parameter_with_log_transform_from_and_to_numerical_zero_is_finite():
+    space = ps.ParameterSpace()
+    space.add(ps.IntegerParameter("p1", bounds=(1, 32), transformation="log"))
+    config = space.from_numerical(np.array([0.0]))
+    vector = space.to_numerical(config)
+    assert np.isfinite(vector).all(), vector
+    assert vector[0] == 0.0
```

Comment:
This currently fails because [np.around](https://github.com/boschresearch/parameterspace/blob/main/parameterspace/transformations/log_zero_one.py#L82) produces a `0` which on transforming back suffers the power of the logarithm and results in `-inf`.

Reply:
Thanks @LGro for finding this bug. I added a commit that hopefully fixes the issue. I also took the liberty to change the tests you introduced slightly to make it more explicit what the root cause was. Now it is hopefully more transparent.
Not sure how we want to handle the PR review now, as we both contributed...any suggestions?